_unittest/test_00_EDB.py

```diff
@@ -2034,6 +2034,12 @@ def test_A118_edb_create_port(self):
                 prim_1_id, ["-60mm", "-4mm"], prim_2_id, ["-59mm", "-4mm"], "port_hori", 30, "Lower"
             )
             assert edb.core_hfss.get_ports_number() == 2
+            port_ver = edb.core_hfss.ports["port_ver"]
+            assert port_ver.hfss_type
+            assert port_ver.horizontal_extent_factor
+            assert port_ver.vertical_extent_factor
```

Comment:
```suggestion
            assert isinstance(port_ver.horizontal_extent_factor, float)
            assert isinstance(port_ver.vertical_extent_factor, float)
```
@ring630 I am not sure but here we could even check the value of the float returned.